Allow custom sidebars with sidebars.js (#62)
UpliftGames/moonwave: website/docs/StaticFiles.md

sidebar_position: 5
---

> # Static Files and Custom Configurations

Reviewer: ```suggestion
# Static Files
```

  --ifm-color-primary-lightest: rgb(146, 224, 208);
  --ifm-code-font-size: 95%;
}
```

## Custom Docs Sidebar

> You can create a custom `.moonwave/sidebars.js` file to set up a custom layout for your `docs` pages. This allows you to specifiy ordering, sections, and even if certain files are excluded.

Reviewer: ```suggestion
You can create a custom `.moonwave/sidebars.js` file to set up a custom ordering for your `docs` sidebar. This allows you to specify ordering, sections, and if certain files are excluded.
```

Typo: I recommend installing https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

> Example `sidebar.js` file:

Reviewer: Is it `sidebar.js` or `sidebars.js`?

```js
module.exports = {
  mySidebar: [
    {
        type: "doc",
        id: "getting-started"
        label: "Getting Started"
    },
    {
      type: 'category',
      label: 'Moonwave',
      items: ['moonwave-basics, moonwave-advances'],
    },
    {
      type: 'category',
      label: 'Other Resources',
      items: ['nested-folder/extra-resources', 'another-folder/even-more-resources'],
    },

>   ],

Reviewer: Can you format this snippet with prettier in a standalone javascript file and then paste it back here?

};
```

> Additional information on customizing your `docs` sidebar can be found at [the Docusaurus Documentation](https://docusaurus.io/docs/sidebar)

Reviewer: ```suggestion
Additional information on customizing your `docs` sidebar can be found at [the Docusaurus Documentation](https://docusaurus.io/docs/sidebar).
```

> Additional information on customizing your `docs` sidebar can be found at (the Docusaurus Documentation)[https://docusaurus.io/docs/sidebar]

Reviewer: ```suggestion
Additional information on customizing your `docs` sidebar can be found at [the Docusaurus Documentation](https://docusaurus.io/docs/sidebar)
```

This link syntax is backwards
